pool/test/assignment_test.ml

```diff
@@ -36,6 +36,27 @@ let confirmation_email contact =
     }
 ;;
 
+let update_contact_event
+  ?increment_assignments_by
```

Comment:
Add a default value for increment and decrement (e.g. 1)
```
  (* argument *)
  ?(increment_assignments_by = 1)

  (* map_or of fold_left *)
  CCOption.map_or ~default:num_assignments (fnc num_assignments)
```

KISS: refactor the function, that it takes one `int` as value with default `0` and uses increment when `> 0` and decrement when `< 0`.